flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/catalog/CatalogTableITCase.scala

```diff
@@ -448,6 +448,55 @@ class CatalogTableITCase(isStreamingMode: Boolean) extends AbstractTestBase {
     assertEquals(expected.sorted, TestCollectionTableFactory.RESULT.sorted)
   }
 
+  @Test
+  def testComputedColumnWithEscapedKeywordField(): Unit = {
```

Comment:
We don't need to introduce a whole new IT case for this, we can just update the existing `testInsertSourceTableWithUserDefinedFuncField` to have a keyword field. IT case is expensive and should be avoid as much as possible.